components/Starknet/modules/architecture_and_concepts/pages/Contracts/contract-storage.adoc

```diff
@@ -16,38 +16,44 @@ The basic function for writing to storage writes value to key is:
 
 [tabs]
 ====
-Cairo 0::
+
+Cairo 1.0::
 +
 [source,js]
 ----
-storage_write(key, value);
+storage_write_syscall(address_domain, address, value)
 ----
-Cairo 1.0::
+
+Cairo 0::
 +
 [source,js]
 ----
-storage_write_syscall(address_domain, address, value)
+storage_write(key, value);
 ----
+
 ====
 // todo add description explaining what address_domain is
 
 Both are system calls that can be imported by adding the line:
```

Comment:
Why does it say "both"? Only `storage_write_syscall` is mentioned. Should `storage_read_syscall` also be mentioned?